src/components/v2/V2Project/index.tsx

```diff
@@ -74,11 +79,16 @@ export default function V2Project({
 
     if (exceptionProjectIds.includes(projectId)) return false
 
-    return ETHBalance?.eq(0)
+    // disable if there's no current funding cycle
+    return !hasCurrentFundingCycle
   }
 
   return (
     <Space direction="vertical" size={GUTTER_PX} style={{ width: '100%' }}>
+      {!hasCurrentFundingCycle && canReconfigureFundingCycles ? (
```

Comment:
could we also only show the `V2BugNotice` if `!hasCurrentFundingcycle` ?

Reply:
totally

Reply:
done